### PR TITLE
Mod filter label

### LINKED ISSUE #446: Change filter state label in each timeline screens
## Kind (Required)

- Improvement

 ## Overview (Required)

- Change the label depending on whether you are filtering.

 ## Screenshot	
Not filtered | Filtering
:--: | :--:
 |

### PR DESCRIPTION
**Pull request by a25max**
## Issue
- close #446

## Overview (Required)
- Show label text if filer now or not filter
- change japanese language file because this file is not functioning.

## Links
- None

## Screenshot
Not  Filter | Filter Now
:--: | :--:
 |

### CODE REVIEW

**Comment by takahirom** [comment]
Thanks! 
Please fix conflict 🙏

**Comment by a25max** [comment]
I resolved conflict.

File: feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/BottomSheetDaySessionsFragment.kt [hunk 1]
```diff
@@ -129,6 +129,7 @@ class BottomSheetDaySessionsFragment : DaggerFragment() {
                 count as Int
             )
             binding.filteredSessionCount.isVisible = uiModel.filters.isFiltered()
+            binding.startFilter.text = if (uiModel.filters.isFiltered()) getString(R.string.filter_now) else getString(R.string.start_filter)
```
Comment (takahirom) [inline feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/BottomSheetDaySessionsFragment.kt]: How about this?
val startFilterTextRes =  if (uiModel.filters.isFiltered()) R.string.filter_now else R.string.start_filter
binding.startFilter.text = getString(startFilterTextRes)

Comment (a25max) [reply feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/BottomSheetDaySessionsFragment.kt]: Good, I fix this.

File: feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/BottomSheetDaySessionsFragment.kt [hunk 2]
```diff
@@ -129,6 +129,8 @@ class BottomSheetDaySessionsFragment : DaggerFragment() {
                 count as Int
             )
             binding.filteredSessionCount.isVisible = uiModel.filters.isFiltered()
+            val startFilterTextRes = if (uiModel.filters.isFiltered()) R.string.filter_now else R.string.start_filter
```
[the remark this answers is not in the document]
Comment (takahirom) [reply feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/BottomSheetDaySessionsFragment.kt]: Can you use {} and newlines?🙏

[the remark this answers is not in the document]
Comment (a25max) [reply feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/BottomSheetDaySessionsFragment.kt]: Oh, Sorry 🙇

**Comment by a25max** [comment]
I have ci test failed. I see https://circleci.com/gh/DroidKaigi/conference-app-2020/569?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

this show me ".circleci/android/danger.bash is failed " only. What I have do?

**Comment by takahirom** [comment]
LGTM！↕️

**Pull request merged**